wildfly/unifiedpush-wildfly-plain/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use latest aerogear/wildfly image as the base
+FROM aerogear/wildfly:2.0.0
```

Comment:
when we inherit from `aerogear/unifiedpush-wildfly:2.0.0` I *think* we only need to do:

* `RUN cp $UPSDIST/servers/unifiedpush-server-wildfly.war $JBOSS_HOME/standalone/deployments` 
* entrypoint

Reply:
But we still need to download and extract the archive and install openssl. Not sure if there is something we can remove.